Update index.md (#56)
uk-tre/website: docs/structure/extending-control.md

# Extending control

## Charter

> This is the main document to refer to, it should contain all of the below and further detail: [Extending control charter](https://www.dropbox.com/scl/fo/fvttq0l9a8i1wfqrcgsdp/AKIK5j3JuXBIM-BFqY6cEAI?rlkey=oke7qkrr8b0vanabs6bg1sq46&dl=0) 

Author: ```suggestion

This is the main document to refer to, it should contain all of the below and further detail: you can find it within this [summary document with information of all working groups](https://docs.google.com/document/d/1uvqALeVixK6PqdkzLFVwDVhc0fPUuBdddyfQzZsRtm4/edit?usp=sharing).

(sorry for the lack of a direct link, but it lives in dropbox)
```

> This is the main document to refer to, it should contain all of the below and further detail: [Extending control charter](https://www.dropbox.com/scl/fo/fvttq0l9a8i1wfqrcgsdp/AKIK5j3JuXBIM-BFqY6cEAI?rlkey=oke7qkrr8b0vanabs6bg1sq46&dl=0 ) 

Author: ```suggestion
This is the main document to refer to, it should contain all of the below and further detail: [Extending control charter](https://www.dropbox.com/scl/fo/fvttq0l9a8i1wfqrcgsdp/AKIK5j3JuXBIM-BFqY6cEAI?rlkey=oke7qkrr8b0vanabs6bg1sq46&dl=0) 
```

With modern IT in the cloud,
with many new tools and architectures becoming mainstream,
allows new options for datasets within a Secure Data Environment in the Data Zone,
to be made available to researchers working in the Research Zone within Trusted Research Environment facilities.  

This Working Groups was set up to support delivery of these overall macro concerns and objectives.

- To minimise Time-to-research and delay
- To make combining datasets easy and reduce barriers to research project scope
- To include Citizen’s agency into the “TRE landscape” and Information Governance.

## Co-chairs and contact

**Chair:**  This is Pete Barnsley, Francis Crick Institute, London  peter.barnsley@crick.ac.uk 
**Contact:**  This is Pete Barnsley, Francis Crick Institute, London  peter.barnsley@crick.ac.uk

## How to get involved
(mailing list, slack channel, repositories…)

Mailing list: WG-EXTENDING-CONTROL-UKTRE-COMM@JISCMAIL.AC.UK
- Subscription form https://www.jiscmail.ac.uk/cgi-bin/webadmin?SUBED1=WG-EXTENDING-CONTROL-UKTRE-COMM&A=1 

> [Dropbox](https://www.dropbox.com/scl/fo/kkbgi1qtkdxn6lmymzx8z/AAqKgPeKwACL9jvvmNhXqSQ?rlkey=55zu0tclr9pq52xwxwk0yxnj7&dl=0)

Author: ```suggestion

Dropbox: ask for the link to existing members or Pete Barnsley, you can also find it within this you can find it within this [summary document with information of all working groups](https://docs.google.com/document/d/1uvqALeVixK6PqdkzLFVwDVhc0fPUuBdddyfQzZsRtm4/edit?usp=sharing). 
```

# Extending control

## Charter

> 

Author: ```suggestion
```

This is the main document to refer to, it should contain all of the below and further detail: you can find it within this [summary document with information of all working groups](https://docs.google.com/document/d/1uvqALeVixK6PqdkzLFVwDVhc0fPUuBdddyfQzZsRtm4/edit?usp=sharing).

(sorry for the lack of a direct link, but it lives in dropbox)

## Overview

> The Extending Control Working group is focused on documenting and outlining possible ways in which data can be ‘shared’ across organisations - when it is not one organisation providing all parts (i.e. the data and trusted research environment).

Author: ```suggestion

The Extending Control Working group is focused on documenting and outlining possible ways in which data can be ‘shared’ across organisations - when it is not one organisation providing all parts (i.e. the data and trusted research environment).
```

With modern IT in the cloud,
with many new tools and architectures becoming mainstream,

> allows new options for datasets within a Secure Data Environment in the Data Zone,
> to be made available to researchers working in the Research Zone within Trusted Research Environment facilities.  

Author: ```suggestion
allows new options for datasets within a Secure Data Environment in the Data Zone to be made available to researchers working in the Research Zone within Trusted Research Environment facilities.  
```

This Working Groups was set up to support delivery of these overall macro concerns and objectives.

- To minimise Time-to-research and delay
- To make combining datasets easy and reduce barriers to research project scope
- To include Citizen’s agency into the “TRE landscape” and Information Governance.

## Co-chairs and contact

> **Chair:**  This is Pete Barnsley, Francis Crick Institute, London  peter.barnsley@crick.ac.uk 

Author: ```suggestion
**Chair:** This is Pete Barnsley, Francis Crick Institute, London peter.barnsley@crick.ac.uk 
```

> **Contact:**  This is Pete Barnsley, Francis Crick Institute, London  peter.barnsley@crick.ac.uk

Author: ```suggestion
**Contact:** This is Pete Barnsley, Francis Crick Institute, London peter.barnsley@crick.ac.uk
```

## How to get involved

> (mailing list, slack channel, repositories…)

Author: ```suggestion

(mailing list, slack channel, repositories…)
```

Mailing list: WG-EXTENDING-CONTROL-UKTRE-COMM@JISCMAIL.AC.UK

> - Subscription form https://www.jiscmail.ac.uk/cgi-bin/webadmin?SUBED1=WG-EXTENDING-CONTROL-UKTRE-COMM&A=1 

Author: ```suggestion

- Subscription form https://www.jiscmail.ac.uk/cgi-bin/webadmin?SUBED1=WG-EXTENDING-CONTROL-UKTRE-COMM&A=1 
```

> 

Author: ```suggestion
```

Dropbox: ask for the link to existing members or Pete Barnsley, you can also find it within this you can find it within this [summary document with information of all working groups](https://docs.google.com/document/d/1uvqALeVixK6PqdkzLFVwDVhc0fPUuBdddyfQzZsRtm4/edit?usp=sharing). 

> People interested in supporting this work should contact the primary contact or chair, or just add to the slack channel discussions.   The prime contact is Pete Barnsley, Francis Crick Institute, London  peter.barnsley@crick.ac.uk

Author: ```suggestion
People interested in supporting this work should contact the primary contact or chair, or just add to the slack channel discussions.
The prime contact is Pete Barnsley, Francis Crick Institute, London  peter.barnsley@crick.ac.uk
```

# Extending control

## Charter

This is the main document to refer to, it should contain all of the below and further detail: you can find it within this [summary document with information of all working groups](https://docs.google.com/document/d/1uvqALeVixK6PqdkzLFVwDVhc0fPUuBdddyfQzZsRtm4/edit?usp=sharing).

(sorry for the lack of a direct link, but it lives in dropbox)

## Overview

The Extending Control Working group is focused on documenting and outlining possible ways in which data can be ‘shared’ across organisations - when it is not one organisation providing all parts (i.e. the data and trusted research environment).
With modern IT in the cloud,
with many new tools and architectures becoming mainstream,

> allows new options for datasets within a Secure Data Environment in the Data Zone to be made available to researchers working in the Research Zone within Trusted Research Environment facilities. 

Author: ```suggestion
allows new options for datasets within a Secure Data Environment in the Data Zone to be made available to researchers working in the Research Zone within Trusted Research Environment facilities.
```

> allows new options for datasets within a Secure Data Environment in the Data Zone to be made available to researchers working in the Research Zone within Trusted Research Environment facilities.  

Author: ```suggestion
allows new options for datasets within a Secure Data Environment in the Data Zone to be made available to researchers working in the Research Zone within Trusted Research Environment facilities. 
```

This Working Groups was set up to support delivery of these overall macro concerns and objectives.

- To minimise Time-to-research and delay
- To make combining datasets easy and reduce barriers to research project scope
- To include Citizen’s agency into the “TRE landscape” and Information Governance.

## Co-chairs and contact

> **Chair:** This is Pete Barnsley, Francis Crick Institute, London peter.barnsley@crick.ac.uk 
> **Contact:** This is Pete Barnsley, Francis Crick Institute, London peter.barnsley@crick.ac.uk

Author: ```suggestion
**Chair:** This is Pete Barnsley, Francis Crick Institute, London peter.barnsley@crick.ac.uk
**Contact:** This is Pete Barnsley, Francis Crick Institute, London peter.barnsley@crick.ac.uk
```

## How to get involved

(mailing list, slack channel, repositories…)

Mailing list: WG-EXTENDING-CONTROL-UKTRE-COMM@JISCMAIL.AC.UK

> - Subscription form https://www.jiscmail.ac.uk/cgi-bin/webadmin?SUBED1=WG-EXTENDING-CONTROL-UKTRE-COMM&A=1 

Author: ```suggestion
- Subscription form https://www.jiscmail.ac.uk/cgi-bin/webadmin?SUBED1=WG-EXTENDING-CONTROL-UKTRE-COMM&A=1
```

> Dropbox: ask for the link to existing members or Pete Barnsley, you can also find it within this you can find it within this [summary document with information of all working groups](https://docs.google.com/document/d/1uvqALeVixK6PqdkzLFVwDVhc0fPUuBdddyfQzZsRtm4/edit?usp=sharing). 

Author: ```suggestion
Dropbox: ask for the link to existing members or Pete Barnsley, you can also find it within this you can find it within this [summary document with information of all working groups](https://docs.google.com/document/d/1uvqALeVixK6PqdkzLFVwDVhc0fPUuBdddyfQzZsRtm4/edit?usp=sharing).
```

People interested in supporting this work should contact the primary contact or chair, or just add to the slack channel discussions.
The prime contact is Pete Barnsley, Francis Crick Institute, London peter.barnsley@crick.ac.uk

> 
> 

Author: ```suggestion
```

> The prime contact is Pete Barnsley, Francis Crick Institute, London  peter.barnsley@crick.ac.uk

Author: ```suggestion
The prime contact is Pete Barnsley, Francis Crick Institute, London peter.barnsley@crick.ac.uk
```
